src/main/java/steps/TestTypeSteps.java

```diff
@@ -112,4 +112,9 @@ public void checkTestWasNotAdded(String testType, String... defects) {
                 assertThat(testTypePage.checkElementIsNotPresent(defect).isEmpty()).isTrue();
             }
     }
+
+    @Step
+    public void checkTestTypesPageIsLoadedByTitle() {
+        assertThat(testTypePage.checkVehicleDetailsIsPresent().isDisplayed()).isTrue();
```

Comment:
Shouldn't checkVehicleDetailsIsPresent().isDisplayed() be in a method inside a page object?

Reply:
no we should keep this outside. However the method name is wrong. I will changeit